src/NexusMods.App.UI/LeftMenu/Items/ILeftMenuItemViewModel.cs

```diff
@@ -20,4 +20,6 @@ public interface ILeftMenuItemViewModel : IViewModelInterface
     public bool IsActive { get; }
     
     public bool IsSelected { get; }
+    
+    public IReadOnlyList<IContextMenuItem>? AdditionalContextMenuItems { get; }
```

Comment:
Does this need to be nullable? Should this not be initialized to empty collection instead?